### PR TITLE
JAMES-3680 Ticket capability should use resolved URL

### PR DESCRIPTION
**Pull request by chibenwa**
Requires https://github.com/apache/james-project/pull/807

### CODE REVIEW

File: tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/method/CustomMethod.scala [hunk 1]
```diff
@@ -31,7 +31,13 @@ case object CustomCapability extends Capability {
 
 class CustomCapabilitiesModule extends AbstractModule {
   @ProvidesIntoSet
-  private def capability(): Capability = CustomCapability
+  private def capability(): CapabilityFactory = CustomCapabilityFactory
+}
+
+case object CustomCapabilityFactory extends CapabilityFactory {
```
Comment (Arsnael) [inline tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/method/CustomMethod.scala:37]: It's me or the `CapabilityFactory` is missing in the PR???

Comment (Arsnael) [reply tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/method/CustomMethod.scala:37]: Ok hard to get back to it, you precised well the apache PR was required...

**Review by Arsnael — APPROVED** [review]

**Review by vttranlina — APPROVED** [review]

File: tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraShortLivedTokenRoutesContract.scala [hunk 2]
```diff
@@ -291,8 +291,8 @@ trait LinagoraShortLivedTokenRoutesContract {
       .queryParam("type", "shortLived")
       .queryParam("deviceId", "abc xyz")
     .when()
-      .basePath(BASE_PATH)
-      .get()
+      .basePath(BASE_PATH).log().all(true)
+      .get().prettyPeek()
```
Comment (vttranlina) [inline tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraShortLivedTokenRoutesContract.scala:295]: debug

File: tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/jwt/ShortLivedTokenRoutes.scala [hunk 3]
```diff
@@ -127,23 +128,26 @@ class ShortLivedTokenRoutes @Inject()(@Named(LongLivedTokenInjectKeys.JMAP) val
...
-    case _: ForbiddenException =>
-      respondDetails(response,
-        ProblemDetails(status = FORBIDDEN, detail = "Using token other accounts is forbidden"),
-        FORBIDDEN)
-    case e =>
-      respondDetails(response,
-        ProblemDetails(status = INTERNAL_SERVER_ERROR, detail = e.getMessage),
-        INTERNAL_SERVER_ERROR)
+  private def handleException(throwable: Throwable, response: HttpServerResponse): SMono[Unit] = {
+    throwable.printStackTrace()
```
Comment (vttranlina) [inline tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/jwt/ShortLivedTokenRoutes.scala:132]: debug

**Pull request merged**